app/src/main/java/ch/epfl/sdp/drone3d/model/mission/SphereToPlaneProjector.kt

```diff
@@ -50,6 +50,7 @@ data class SphereToPlaneProjector(val origin: LatLng) {
     fun toLatLng(point: Point): LatLng {
 
         var longitude = point.x * meterToLong + origin.longitude
+        //PBM: if origin at -179 and add lots of meters => it should go to 179 and not -177
```

Comment:
You should either correct the code or remove the comment. And I think that it should indeed go to -177 first (and then to 179 if you really add a gigantic distance that goes all around the earth, but wouldn't make much sense for our application). It would go to 179 first if you add negativ values

Reply:
Thanks for the help correcting